src/confcom/azext_confcom/README.md

```diff
@@ -670,7 +670,7 @@ For information on what a policy fragment is, see [policy fragments](#policy-fra
 Example 1: The following command creates a security fragment and prints it to stdout as well as saving it to a file `contoso.rego`:
 
 ```bash
-az confcom acifragmentgen --config ./fragment_config.json --svn 1 --namespace contoso
+az confcom acifragmentgen --input ./fragment_config.json --svn 1 --namespace contoso
```

Comment:
would --fragment make more sense?

Reply:
`--input` is the same for policy and fragment generation, so this is consistent across the different commands